sql/analyzer/analyzer.go

```diff
@@ -436,13 +445,16 @@ func postPrepareRuleSelector(id RuleId) bool {
 		pushdownSortId,
 		pushdownGroupbyAliasesId,
 		qualifyColumnsId,
+		//pruneTablesId,
 		resolveColumnsId,
 		resolveColumnDefaultsId,
 		expandStarsId,
+		//reorderProjectionId,
 
 		// OnceAfterDefault
 		pushdownFiltersId,
 		subqueryIndexesId,
+		//eraseProjectionId,
```

Comment:
1) all 3 of these are pre-only, cut from post?
2) delete commented lines if you're cutting

Reply:
1. These are all things I thought needed to be done post, but I thought wrong.
2. ok